src/libraries/System.Private.CoreLib/src/System/Globalization/CultureInfo.cs

```diff
@@ -750,6 +750,9 @@ public virtual DateTimeFormatInfo DateTimeFormat
         public void ClearCachedData()
         {
             // reset the default culture values
+#if TARGET_WINDOWS
+            s_userDefaultLocaleName = null; // this is set by GetUserDefaultCulture();
```

Comment:
>null [](start = 38, length = 4)

we should just initialize it here. Also we should initialize it at its definition.